shared/profile/showcased-team-info/container.js

```diff
@@ -18,11 +18,8 @@ const mapStateToProps = (state: TypedState, {routeProps}) => {
   const memberCount = team.numMembers
   const openTeam = team.open
   const teamname = team.fqName
-  const youAreInTeam = !!state.entities.getIn(['teams', 'teamnames', teamname], false)
-  const youHaveRequestedAccess = !!state.entities.getIn(
-    ['teams', 'teamAccessRequestsPending', teamname],
-    false
-  )
+  const youAreInTeam = !!state.teams.getIn(['teamnames', teamname], false)
```

Comment:
Actually a helper already exists (which I use in #11290). Should that helper be fixed?

Reply:
Okay, I fixed it, but didn't want to lump it in with this PR since this is supposed to be a refactoring PR. It's in https://github.com/keybase/client/pull/11326